README.md

```diff
@@ -1 +1,18 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# Greetings!  
+
+My name is **Maya Krasovsky**.
+
+![*This is me outisde, taking in the sunshine*](/Users/mayakrasovsky/Desktop/IMG_8762.jpg)
+
```

Comment:
Your image did not upload in the README because it is stored in your local machine in the Desktop directory. Even though you could see your image data when you committed to github, when I try to access your repo from my machine, I do not have permission to see your local files. I had the same issue and resolved it by putting my image inside my hw01 repo (Users/kevinsong/Desktop/<img> vs Users/kevinsong/Desktop/hw01/<img>), that way the image is counted as a staged file when I commit everything in my repo.

Reply:
Thanks! I fixed the problem with the picture in the newest commit